src/modules/RokulRunnings.ts

```diff
@@ -57,14 +57,35 @@ export class RokulRunnings {
   /** Launches the channel corresponding to the specified channel ID. */
   launchTheChannel({
     channelCode,
+    contentId,
+    mediaType,
     retries = this.retries,
     params,
   }: {
     channelCode: string;
+    contentId?: string;
+    mediaType?: string;
     retries?: number;
     params?: Params;
   }) {
-    return this.driver.sendLaunchChannel({ channelCode, retries, params });
+    return this.driver.sendLaunchChannel({ channelCode, contentId, mediaType, retries, params });
+  }
+
+  /** Deep Links into channel */
+  deepLinkIntoChannel({
+    channelCode,
+    contentId,
+    mediaType,
+    retries = this.retries,
+    params,
+  }: {
+    channelCode: string;
+    contentId?: string;
+    mediaType?: string;
+    retries?: number;
+    params?: Params;
+  }) {
+    return this.driver.deepLink({ channelCode, contentId, mediaType, retries, params });
```

Comment:
do you need to `await` this ?

Reply:
I think the return takes care of the await? (At least, that's what VSCode tells me...)

Reply:
`async` functions will effectively return a Promiise, so it will have to get handled at some point.